test/Project.toml

```diff
@@ -8,6 +8,8 @@ MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+StartUpDG = "472ebc20-7c99-4d4b-9470-8fde4e9faa0f"
+StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
```

Comment:
You should add some `[compat]` bounds

Reply:
Done. The compat bounds for StructArrays are set to 0.6, which should be the next tagged release.